intro/gui.rst

```diff
@@ -50,6 +50,20 @@ Error
 	prevent data loss e.g. when the folder path was unmounted. If the marker
 	was deleted accidentally, just recreate it and press the rescan button
 	in the UI.
+	
+Creating new synced Folders
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the Syncthing web interface. Click the "Add Folder" button:
+
+Set the folder label to something descriptive. The label you enter here will be shared with remote devices, but can be changed on each device, as desired.
+
+The folder id has to be the same across all devices, it is the unique identifier of the folder. Best practice is to keep the auto-generated id to avoid conflicts with equally named folders on other devices.
+
+The folder path designates the physical path to the folder, i.e. on your hard drive.
+
+Next select with which devices to share and you are good to go.
```

Comment:
to share the folder with and press add, which will add the folder and start syncing.